website/docs/cdktf/concepts/hcl-interoperability.mdx

```diff
@@ -224,6 +226,8 @@ output "name" {
 }
 ```
 
+
+
 ## HCL to CDKTF
 
 Use the [`cdktf convert`](/terraform/cdktf/cli-reference/commands#convert) command to automatically translate existing HCL into a preferred CDKTF language. The [modules documentation](/terraform/cdktf/concepts/modules) shows how to use existing Terraform modules in CDK for Terraform projects.
```

Comment:
I know you didn't write this bit but now that I'm looking at this it seems really awkward. I think this should be a bulleted list to denote that these are two different ways to interact with HCL in your CDK project.